server/paymentProviders/opencollective/host.js

```diff
@@ -57,6 +65,9 @@ paymentMethodProvider.processOrder = async order => {
     platformFeeInHostCurrency,
     paymentProcessorFeeInHostCurrency: 0,
     description: order.description,
+    data: {
+      isFeesOnTop: order.data?.isFeesOnTop,
```

Comment:
Should ideally be a boolean, instead of being loosely false (null/undefined).

Reply:
Done. 👍🏽